tools/gen_internal_conf/src/main.rs

```diff
@@ -286,6 +293,28 @@ fn parse_memory_size(mem_str: &str) -> Result<usize, &str> {
     Ok(mem_val * unit_factor)
 }
 
+fn get_u64_id_high_and_low(id: &OcclumMetaID) -> (u64, u64) {
+    let id_high = u64::from_str_radix(id.high.trim_start_matches("0x"), 16).expect("Hex string ID required");
```

Comment:
Ideally, we could use the parser to get the value instead of get it from string.